lib/Makefile.am

```diff
@@ -22,3 +22,18 @@ if BUILT_IN_PYTHON_PLUGIN_SUPPORT
 libbabeltrace_la_LIBADD += $(top_builddir)/python-plugin-provider/libbabeltrace-python-plugin-provider.la
 endif
 
+# Build a libbabeltrace-ctf library for backwards compatibility.
+# CTF writer used to be in libbabeltrace-ctf in Babeltrace 1, so this
+# file must still exist. As of Babeltrace 2, CTF writer is implemented
+# in libbabeltrace.
+libbabeltrace_ctf_la_SOURCES = babeltrace.c values.c ref.c logging.c
+libbabeltrace_ctf_la_LDFLAGS = $(LT_NO_UNDEFINED) \
+			-version-info $(BABELTRACE_LIBRARY_VERSION)
+
+libbabeltrace_ctf_la_LIBADD = \
```

Comment:
I think we never shipped a CTF writer dll, so we could just forego this lib entirely on Windows.